en/docs/install-and-setup/setup/api-controller/building-jenkins-ci-cd-pipeline-apictl.md

```diff
@@ -0,0 +1,328 @@
+
+    <a href="{{base_path}}/assets/img/learn/api-controller/webhook-trigger-plugin.png"><img src="{{base_path}}/assets/img/learn/api-controller/webhook-trigger-plugin.png" alt="" width="500"></a>
+
+6. Configure API CTL as an environment variable. Go to the **Manage Jenkins** section and select **Configure System**. 
+    <a href="{{base_path}}/assets/img/learn/api-controller/configure-system.png"><img src="{{base_path}}/assets/img/learn/api-controller/configure-system.png" alt="" width="500"></a>
+
+    Under the **Global properties** select **Environment variables** and set the APICTL home as below and click **Save**.
+    <a href="{{base_path}}/assets/img/learn/api-controller/environment-variables.png"><img src="{{base_path}}/assets/img/learn/api-controller/environment-variables.png" alt="" width="70%"></a>
+
+
+Now the Jenkins server is set up.
+
+### Step 2 - Create Github repositories
+
+In this section, two repositories are created to illustrate this scenario.
+
+1. Source Repository -[https://github.com/chamilaadhi/poc-cicd-source-repo](https://github.com/chamilaadhi/poc-cicd-source-repo)
+
+2. Deployment Repository - [https://github.com/chamilaadhi/poc-cicd-deployment-repo](https://github.com/chamilaadhi/poc-cicd-deployment-repo)
+
+Source repository contains the files and metadata related to the API. Deployment repository contains the configurations related to each environment this APIs is getting deployed. 
+
+#### Step 2.1 - Setup source repository
+
+This section shows how to trigger a Jenkins job when a change is made to the repository. Lets use [Github webhooks](https://docs.github.com/en/developers/webhooks-and-events/webhooks) to trigger a Jenkins job. 
+
+1. Go to the **source** repository and under the **Settings**, select **Webhooks** and add a webhook to the jenkins server.
+
+  <a href="{{base_path}}/assets/img/learn/api-controller/select-webhooks.png"><img src="{{base_path}}/assets/img/learn/api-controller/select-webhooks.png" alt="" width="70%"></a>
+
+2. Use your jenkins server URL (e.g., `http://&lt;public_ip>:8080/github-webhook/`) as the URL and **Content type** as application/json.
+
+3. Under the **Which events would you like to trigger this webhook?** section, select **Let me select individual events**, and select **pull request** and **pushes**.
+
+  <a href="{{base_path}}/assets/img/learn/api-controller/jenkins-server-url.png"><img src="{{base_path}}/assets/img/learn/api-controller/jenkins-server-url.png" alt="" width="70%"></a>
+
+Source repository is now configured. 
+
+Now let’s initialize the source repository. This will be done from the developer’s local machine.
+
+!!! note
+    If you haven’t setup API Controller, Please set it up using [https://github.com/wso2/product-apim-tooling/releases/tag/v4.0.0](https://github.com/wso2/product-apim-tooling/releases/tag/v4.0.0) 
```

Comment:
Let's not use that URL here too. Instead use https://apim.docs.wso2.com/en/4.0.0/install-and-setup/setup/api-controller/getting-started-with-wso2-api-controller/#download-and-initialize-the-apictl

Reply:
Done